### PR TITLE
Create and style desktop version of the website

### PR DESCRIPTION
**Pull request by Andyveloper**
## Changes
- Create and style desktop version
- Create Media queries for 768px and up
- Create Media queries for 1600px and up
- Add new divs and new grids to make it easier to style
- Increase font-size at 1600px and up

## Visualization recommended settings
- Please visualize it at 1440px, since this is the size of the FIGMA template and therefore, its the way it looks the most similar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,6 +18,14 @@
...
     <nav>
       <img id="burgermenu" src="./img/burger.png" alt="">
+      <div id="menu">
+        <ul id="menu-container">
+          <li class="menu-item"><a class="menu-style" href="#">Portfolio</a></li>
+          <li class="menu-item"><a class="menu-style" href="#">About</a></li>
+          <li class="menu-item"><a class="menu-style" href="#">Contact</a></li>
+          <li class="menu-item"><img id="mail-icon" src="/img/icons/icons8-mail-48.png" alt=""></li>
+        </ul>
+      </div>
```
Comment (youmari) [inline index.html]: you have a gap in 800px widescreen, I suggest fixing that so your portfolio will display on any device smoothly
image

Comment (youmari) [inline index.html]: Your headline section doesn't look as the one the Figma Design I see you are using the `height: 100vh;` that cause this please try to adjust the style in order that you page look the same as the one in the Figma design 

- [ ] Yours
image

- [ ] Figma design 
image

File: index.html [hunk 2]
```diff
@@ -18,6 +18,14 @@
...
     </nav>
   </header>
```
Comment (youmari) [inline index.html:31]: also, you forgot to include the illustrations in the headline section I suggest doing that 
image

File: index.html [hunk 3]
```diff
@@ -44,19 +52,24 @@ <h3 id="subhero">I am a Software Developer</h3>
...
+          <h2 id="subtitle">Multi-Post Stories</h2>
+          <p id="text-content">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <ul id="lang-boxes">
+            <li class="lang-box1">css</li>
+            <li class="lang-box1">html</li>
+            <li class="lang-box1">bootstrap</li>
+            <li class="lang-box1">Ruby</li>
+          </ul>
+          <button class="main-button">See Project</button>
+        </div>
```
Comment (youmari) [inline index.html]: Kindly try to fix your image src link right now I can't see them you can fix them by adding a dot before the slash and use the right path
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Zeraltz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter errors
✔️ Descriptive Readme

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Zeraltz,

Good job implementing all the changes asked by the previous reviewer. The project looks great 💪💪💪

ClappingLeonardoDicaprioGIF (2)

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**